src/KafkaFlow.Abstractions/IMessageDecompressor.cs

```diff
@@ -0,0 +1,15 @@
+namespace KafkaFlow
+{
+    /// <summary>
+    /// Used to create a message decompressor
+    /// </summary>
+    public interface IMessageDecompressor
```

Comment:
Should we follow the same "naming strategy" used by the Serializer just calling IDecompressor and ICompressor?

Reply:
I think we should, the compression and decompression contract should not be attached to the message or any other object IMO

Reply:
Renamed to `IDecompressor` and `ICompressor`